src/gui/screen_printing.cpp

```diff
@@ -222,18 +221,21 @@ void screen_printing_init(screen_t *screen) {
     window_hide(id);
     pw->message_flag = 0;
 
+    // buttons
+    const uint16_t icon_y = gui_defaults.footer_sz.y - gui_defaults.padding.bottom - 22 - 64;
```

Comment:
64 and 22 should be constants too

Reply:
They should but I don't know what they represent. Or it needs another refactor through the code.